common/api/core-common.api.md

```diff
@@ -331,6 +331,75 @@ export namespace AreaPattern {
     }
 }
 
+// @public
+export namespace AtmosphericScattering {
+    export interface Props {
+        atmosphereHeightAboveEarth?: number;
+        brightnessAdaptationStrength?: number;
+        densityFalloff?: number;
+        inScatteringIntensity?: number;
+        minDensityHeightBelowEarth?: number;
+        numInScatteringPoints?: number;
+        numOpticalDepthPoints?: number;
+        outScatteringIntensity?: number;
+        scatteringStrength?: number;
+        wavelengths?: WavelengthsProps;
+    }
+    export class Settings implements Props {
+        // (undocumented)
+        readonly atmosphereHeightAboveEarth: number;
+        // (undocumented)
+        readonly brightnessAdaptationStrength: number;
+        // (undocumented)
+        static readonly defaults: Required<Props>;

```

Comment:
Why is this a `Props` instead of a `Settings`?

Reply:
The contents of `defaults` are [used within the `Settings` constructor](https://github.com/iTwin/itwinjs-core/blob/97e4bb491bf84c16f90bd3ce6cf0f2de86a03222/core/common/src/AtmosphericScattering.ts#L133-L146) when no value is provided.
`Settings` are built from a `Props` argument, so the default arguments are held as the same type.

Reply:
I understand that, but it deviates from the pattern established elsewhere, e.g., see AmbientOcclusion, Environment, FeatureAppearance, GroundPlane, etc. These are also immutable, unlike `Required<Props>`.

Reply:
Oh, `defaults` is a public property. I see why the immutability of `Settings` is desired now.